app/src/main/java/org/fossasia/pslab/activity/MainActivity.java

```diff
@@ -385,6 +393,8 @@ public void onReceive(Context context, Intent intent) {
                             Toast.makeText(getApplicationContext(), getString(R.string.device_connected_successfully), Toast.LENGTH_SHORT).show();
                             if (navItemIndex == 0) {
                                 getSupportFragmentManager().beginTransaction().replace(R.id.frame, InstrumentsFragment.newInstance()).commit();
+                            }else if(navItemIndex == 1){
+                                getSupportFragmentManager().beginTransaction().replace(R.id.frame,HomeFragment.newInstance(true,true)).commitAllowingStateLoss();
```

Comment:
May be you forgot to change back `HomeFragment.newInstance(true,true)` to its original variables?

Reply:
I think thatpart is handled in USB Detach Receiver.

Reply:
Yea got it! :+1: